### PR TITLE
Ahu cloudinary

### PR DESCRIPTION
**Pull request by AvSum1006**
- add cloudinary
- integrate image dragons on show + index
- integrate profile image navbar + sign up

### CODE REVIEW

File: app/models/dragon.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class Dragon < ApplicationRecord
   CATEGORY = %w[Wyvern Wyrm Quetzalcoatl Hydra Drake]
 
+  has_one_attached :image
+
```
Comment (ngrognet) [inline app/models/dragon.rb]: remove blank line

File: app/views/dragons/index.html.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@
       <div class="col-12 col-sm-6 col-lg-3 my-2">
           <%= link_to dragon_path(dragon), style: "text-decoration: none; color: black" do %>
           <div class="border rounded shadow" style="width: auto; height: 500px;">
-            <%= image_tag("smaug.webp", class: "card-img-top rounded", style: "height: 300px; object-fit:cover") %>
+            <%= cl_image_tag dragon.image.key, class: 'w-100 card-img-top rounded', style: 'height: 300px; object-fit: cover;' %>
```
Comment (ngrognet) [inline app/views/dragons/index.html.erb]: why the need for w-100 now, wasn't it working before?

**Review by ngrognet — CHANGES_REQUESTED** [review]
just left a few small comments, but looks ready for merge :)

File: app/models/dragon.rb (hunk 1 above)
Comment (AvSum1006) [reply app/models/dragon.rb]: Done

File: app/views/dragons/index.html.erb (hunk 2 above)
Comment (AvSum1006) [reply app/views/dragons/index.html.erb]: You're right, I removed it !

**Pull request merged**